frontend/src/static/sidebar.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "Main activities",
+        "items": [
+            {
+                "name": "Dashboard",
+                "icon": "DashboardIcon",
+                "link": "/dashboard",
+                "disabled": false
+            },
+            {
+                "name": "Grant Access",
+                "icon": "VirtualColumnKeyIcon",
+                "link": "/grant",
+                "disabled": false
+            }
+        ]
+    },
+    {
+        "name": "Management",
+        "items": [
+            {
+                "name": "Settings",
+                "icon": "SettingsIcon",
+                "link": "#",
```

Comment:
This **"Management"** section does not seem to be implemented.
Should comment it out or remove it, unless it's been discussed to leave it there.